### DISCUSSION TITLE
Using toggle list for user cases in DG

### DISCUSSION THREAD

**Discussion by c0j0s** [AY2223S2-CS2103-F11-2/tp#24] [General]
AB3 uses standard markdown for writing use cases.
It is fine for a limited number of use cases, but based on our feature plan, we should have more than a dozen of these.

I think we should use a more concise format for organizing our use cases.
like a toggle list that we can collapse when browsing the DG:
```
<details>
<summary><b>[DTM1] Add a delivery task</b></summary>
<pre>
<b>MSS</b>
...
<b>Extensions</b>
...
</pre>
</details>  
```
that look like this
<details>
<summary><b>[DTM1] Add a delivery task</b></summary>
<pre>
<b>MSS</b>
...
<b>Extensions</b>
...
</pre>
</details>  

what you guys think?

**Comment by dohaduong** [comment]
Oh yeap agree, i think it's better to use toggle list.
